doc/development/development_on_osx.rst

```diff
@@ -89,16 +89,17 @@ Other Packages
 
 There are a bunch of other packages that can easily be installed using pip and brew:
 
-.. code-block:: none
+.. code-block:: bash
 
     brew install homebrew/python/pillow gmp mpfr libmpc libsodium
     sudo easy_install pip
     pip install --user cython  # Needs to be installed first for meliae
-    pip install --user cherrypy cffi chardet configobj cryptography decorator gmpy2 idna meliae netifaces numpy pillow psutil pyasn1 pycparser scipy twisted service_identity libnacl bitcoinlib
+    pip install --user cherrypy cffi chardet configobj cryptography decorator gmpy2 idna meliae netifaces numpy pillow \
+    psutil pyasn1 pycparser scipy lz4 pyopenssl Twisted==16.4.1 networkx service_identity libnacl bitcoinlib
```

Comment:
Can we put this into a __requirements.txt__ file (maybe with pegged versions?) and advocate __pip install -r requirements.txt__ instead?